### PR TITLE
feat(bookings): Modified booking schema + Added new booking creation flow

### PR DESCRIPTION
**Pull request by paradoux**
- Created migrations to modify bookings schema
- Added route to create booking
- Created validation functions to check new booking data
- Created handler to insert new booking to db following db keys naming

### CODE REVIEW

File: packages/server/src/utils/booking.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    departure_time: newBookingData.departureTime,
+    comments: newBookingData.comments,
+    companions: newBookingData.companions,
+    booking_status: "pending" as BookingStatus,
+  };
+
+  await insertNewBooking(formattedBooking);
+};
+
+export const validateNewBookingData = async (
```
Comment (AlexGeb) [inline packages/server/src/utils/booking.ts:34]: Why not using something lie yup to validate ?

File: packages/server/src/db/bookings.ts [hunk 2]
```diff
@@ -1,13 +1,26 @@
-import { Booking } from "../domain/Booking";
-
 import { knex } from "./db";
 
-export const getAllBookings = async () => {
-  const bookings = await knex<ReadonlyArray<Booking>>("bookings").select();
+export type BookingStatus = "pending" | "accepted" | "rejected";
```
[the remark this answers is not in the document]
Comment (paradoux) [reply packages/server/src/db/bookings.ts:3]: I think string literals are more encouraged (https://dev.to/bholmesdev/using-enums-with-string-values-in-typescript-consider-string-literals-instead-486e)

File: packages/server/src/db/bookings.ts [hunk 3]
```diff
@@ -1,13 +1,26 @@
...
+
+export const getAllBookings = async (): Promise<Booking[]> => {
+  const bookings = await knex("bookings").select();
   return bookings;
 };
 
-export const getBookingById = async (bookingId: string) => {
-  const booking = await knex<Booking>("bookings").where("id", bookingId);
+export const getBookingById = async (bookingId: string): Promise<Booking> => {
+  const [booking] = await knex("bookings").where("booking_id", bookingId);
```
[the remark this answers is not in the document]
Comment (paradoux) [reply packages/server/src/db/bookings.ts:20]: There's .first() maybe  :))

File: packages/server/src/utils/booking.ts (hunk 1 above)
Comment (paradoux) [reply packages/server/src/utils/booking.ts:34]: So I didn't know about yup but yeah I thought about using joi too, just thought we might want to add more validation logic than just types, like checking the account exists in our db etc so I created a function :))

**Pull request merged**